<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_PlaylistPreferences</class>
 <widget class="QWidget" name="GUI_PlaylistPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>909</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Behavior</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="5" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>828</width>
            <height>836</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Start up</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <item>
               <widget class="QCheckBox" name="cbLoadTemporaryPlaylists">
                <property name="text">
                 <string>Load temporary playlists</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbLoadSavedPlaylists">
                <property name="text">
                 <string>Load saved playlists</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbStartPlaying">
                <property name="text">
                 <string>Start playing</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbLoadLastTrack">
                <property name="text">
                 <string>Load last track on startup</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbRememberTime">
                <property name="text">
                 <string>Remember time of last track</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbFilesystemPlaylistName">
             <property name="title">
              <string>Start up from file manager</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QCheckBox" name="cbCreateFileystemPlaylist">
                <property name="text">
                 <string>Use special playlist when starting Sayonara from file manager</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QCheckBox" name="cbChooseFilesystemPlaylistName">
                  <property name="text">
                   <string>Playlist name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="leFilesystemPlaylistName"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbRememberAfterStop">
             <property name="text">
              <string>Load last track after resuming from stopped state</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbPlayTrackAfterSearch">
             <property name="text">
              <string>Play track when pressing &quot;Enter&quot; in search field</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbJumpToCurrentTrack">
             <property name="text">
              <string>Jump to current track in list when track changes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbConfirmOnClose">
             <property name="text">
              <string>Show confirmation on close</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Locked Playlists</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="cbAllowDynamicPlayback">
         <property name="text">
          <string>Allow dynamic playback</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbAllowRearrange">
         <property name="text">
          <string>Allow re-arrange functionality (e.g. sorting)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Look</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Visible items</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QCheckBox" name="cbShowClearButton">
            <property name="text">
             <string>Show clear button</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowBottomBar">
            <property name="text">
             <string>Show footer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowNumbers">
            <property name="text">
             <string>Show numbers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowCovers">
            <property name="text">
             <string>Show covers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowRating">
            <property name="text">
             <string>Show rating</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Current track</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbCustomColorStandard">
            <property name="text">
             <string>Custom font color in standard theme</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbCustomColorDark">
            <property name="text">
             <string>Custom font color in dark theme</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnCustomColorDark">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnCustomColorStandard">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true">...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Row style</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Row style</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="leExpression"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnDefault">
              <property name="text">
               <string notr="true">Default</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTemplateHelp">
              <property name="text">
               <string>Help</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widgetTemplateHelp" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="8" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Example</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="filenameTitle">
               <property name="text">
                <string notr="true">File name</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string notr="true">%title%</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string notr="true"> %nr% - '%title%' - %artist%</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string notr="true">%filename%</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <italic>true</italic>
                </font>
               </property>
               <property name="text">
                <string>'italic text'</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="labTitle">
               <property name="text">
                <string notr="true">title</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string notr="true">%album%</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="labAlbum">
               <property name="text">
                <string notr="true">album</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string notr="true">%nr%</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labTrackNumber">
               <property name="text">
                <string notr="true">track number</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="labArtist">
               <property name="text">
                <string notr="true">artist</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string notr="true">%artist%</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>*bold text*</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Workarounds</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Workarounds</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="cb_startupPlaybackWorkaround263">
            <property name="text">
             <string notr="true">Fix startup playback issue</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbLoadTemporaryPlaylists</tabstop>
  <tabstop>cbLoadSavedPlaylists</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
